cheri_compressed_cap.h

```diff
@@ -93,6 +98,25 @@ enum CC256_OTypes {
     CC256_SPECIAL_OTYPE(LAST_NONRESERVED_OTYPE, 15),
 };
 
+#define LS_SPECIAL_OTYPES_COMMON(ITEM, ...)                                                                            \
```

Comment:
Loads of cases of having way too much space before the `\`s

Reply:
This is a result of clang-format

Reply:
*Sigh* stupid config

Reply:
Yeah maybe worth dropping adding
```
AlignEscapedNewlines: DontAlign
```
or 
```
AlignEscapedNewlines: Left
```

Don't really care which one, but the right alignment seems like an odd default.

Reply:
The default is fine when the line limit is 72 or 80, but for 120 it gets silly

Reply:
Right is the most visually pleasing IMO, but I can change to AlignEscapedNewlines: Left and reformat everything if any particularly cares. Otherwise, I will leave as is.

Reply:
I don't mind. Leaving it as is is fine.